### ISSUE TITLE
version 1.4.1 always syncing

### ISSUE THREAD

**Issue by z0ow** [niceit/tabby-cloud-sync-settings#21]
![image](https://user-images.githubusercontent.com/20352723/172030415-37b222f1-22db-4170-bebe-acc44ab5e58d.png)

cloud-sync-settings is always alert that synced saved settings (picture) after I upgraded cloud-sync-settings to version 1.4.1. It is unexpected behavior because I am not changing/updating tabby settings.

**Comment by niceit** [comment]
that is old cache of tabby I think, you could open it’s developer tools and clear the cache

**Comment by z0ow** [comment]
> that is old cache of tabby I think, you could open it’s developer tools and clear the cache

Thank you for your attention. It puzzled me because I am not a web application developer. Is it clear the cache means open DevTools then click Application->Storage->Clear Site Data? tabby is still disturbing me with this alert when I focus on a tab. The best solution for me now is to disable cloud-sync-settings 😞

**Comment by niceit** [comment]
Could you provide your current cloud sync? I will take a look

**Comment by z0ow** [comment]
Thanks, what file or information do I need to provide to you?

**Comment by niceit** [comment]
@z0ow you could update to version 1.4.2 and give a try

**Comment by z0ow** [comment]
@niceit hi. thanks, I am sorry to ask an immature question how can I upgrade the version to 1.4.2 outside of tabby? In my tabby plugin manager, the version of cloud-sync-settings is still 1.4.1 whatever I reload it.

**Comment by niceit** [comment]
I'm not sure why you can not upgrade the plugin via Tabby? doing an another unofficial way may cause problem

**Comment by z0ow** [comment]
> I'm not sure why you can not upgrade the plugin via Tabby? doing an another unofficial way may cause problem

Yeah, I downloaded the package, unpacked it, and overwrote the cloud-sync-settings folder under tabby/plugin, but I got another error message. This plugin probably downloads the wrong configuration from the cloud or disorders it.

I am afraid I don't have much time to care about why this version failed to sync configuration because two exams are incoming. I have to downgrade the version from 1.4.x to 1.3.0 manually, and now everything is ok.

Thank you for paid much time to work around my problem. I will provide more information and help test when I am free.
